### PR TITLE
Add functions to create Kueue resources

### PR DESCRIPTION
**Pull request by sutaakar**
# Issue link

# What changes have been made

Add function to create a Kueue LocalQueue.

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: support/kueue.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func CreateKueueLocalQueue(t Test, namespace, clusterQueueName string) *v1beta1.LocalQueue {
+	t.T().Helper()
+
+	localQueue := &v1beta1.LocalQueue{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: corev1.SchemeGroupVersion.String(),
```
Comment (dgrove-oss) [inline support/kueue.go:32]:
```suggestion
			APIVersion: v1beta1.SchemeGroupVersion.String(),
```

File: support/kueue.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			Kind:       "ConfigMap",
```
Comment (dgrove-oss) [inline support/kueue.go:33]:
```suggestion
			Kind:       "LocalQueue",
```

**Review by dgrove-oss — COMMENTED** [review]
This seems useful. but need to fix the GVK if we still want it.

**Comment by tedhtchang** [comment]
@sutaakar It would be handy if I can call `local_q, err = test.Client().Kueue().KueueV1beta1().LocalQueue().create(...)`  in a training operator e2e test

**Comment by sutaakar** [comment]
@tedhtchang I suppose it is done in https://github.com/project-codeflare/codeflare-common/pull/40, right?

**Comment by sutaakar** [comment]
@dgrove-oss Fixed and rebased

**Comment by sutaakar** [comment]
Please take a look

**Review by jiripetrlik — APPROVED** [review]

**Pull request merged**